### PR TITLE
x11-terms/guake: version bump to 3.3.0

### PR DESCRIPTION
**Pull request by cklaucke**
Bug: https://bugs.gentoo.org/644666

### CODE REVIEW

File: x11-terms/guake/guake-3.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+HOMEPAGE="https://github.com/Guake/guake https://pypi.org/project/Guake"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="utempter"
+
+RDEPEND="dev-libs/keybinder:3
+		dev-python/dbus-python[${PYTHON_USEDEP}]
```
Comment (Zlogene) [inline x11-terms/guake/guake-3.3.0.ebuild]: make tabs consistent here

**Comment by cklaucke** [comment]
fixed tabs and added metadata

File: x11-terms/guake/guake-3.3.0.ebuild (hunk 1 above)
Comment (cklaucke) [reply x11-terms/guake/guake-3.3.0.ebuild]: done

File: x11-terms/guake/guake-3.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="utempter"
+
+RDEPEND="
+	dev-libs/keybinder:3
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pbr[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	utempter? ( sys-libs/libutempter )
```
Comment (johu) [inline x11-terms/guake/guake-3.3.0.ebuild]: Please move the optional dependency to the bottom.

Comment (cklaucke) [reply x11-terms/guake/guake-3.3.0.ebuild]: Fixed in c78ea22.

**Comment by johu** [comment]
Thanks

**Pull request closed without merging**